### PR TITLE
use direct import for ajax to avoid bundling whole library. Make rxjs…

### PR DESCRIPTION
**Pull request by Tomekmularczyk**
… a peer dependency.

This resolves (tried it) issue #2.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,11 +29,14 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
+    "rxjs": "^5.5.0",
```
Comment (mcmunder) [inline package.json]: Does it really also have to be declared as a dev dependency? Just asking to learn ;-)

File: package.json [hunk 2]
```diff
@@ -29,11 +29,14 @@
...
     "tap-spec": "^4.1.1",
     "tape": "^4.8.0"
   },
   "dependencies": {
-    "rxjs": "^5.5.0",
     "xhr2": "^0.1.4"
+  },
+  "peerDependencies": {
+    "rxjs": "^5.5.0"
```
Comment (mcmunder) [inline package.json]: shouldn't `"rxjs": "^5.0"` be sufficient? If they stick to [semver](https://semver.org/) breaking changes should come with a change of the major version?! See https://docs.npmjs.com/files/package.json#peerdependencies

File: package.json (hunk 1 above)
Comment (Tomekmularczyk) [reply package.json]: This library is intended to be used as a complimentary package for `rxjs` so we expect that user will have it installed (that's why we declare it as "peerDependency"). However, we need `rxjs` for testing purposes - that's why it's "devDependency".

File: package.json (hunk 2 above)
Comment (Tomekmularczyk) [reply package.json]: I think that's a good remark. Maybe we can just do `"rxjs": "5.x"` like in the link you provided with `tea-latte` example. I will try that and update PR.

**Pull request merged**